src/ixdat/data_series.py

```diff
@@ -235,3 +178,203 @@ def data(self):
                     f"{len(self._data.shape)}-dimensional."
                 )
         return self._data
+
+
+class ValueSeries(Field):
+    """Class to store scalar values that are measured over time.
+
+    Characterized by a reference to the corresponding time series. This reference is
+    represented in relational databases as a row in an auxiliary linker table
+    """
+
+    series_type = "vseries"
+
+    def __init__(
+        self,
+        name,
+        unit_name,
+        data,
+        t_id=None,
+        tseries=None,
+        a_ids=None,
+        axes_series=None,
+    ):
+        """Initiate a ValueSeries with a TimeSeries or a reference thereto
+
+        Args (in addition to those of parent):
```

Comment:
added smething according to [this](https://stackoverflow.com/questions/7753805/sphinx-automodule-how-to-reference-classes-in-same-module), hope it will work.